### ISSUE TITLE
How to use eventstore_exporter with config file?

### ISSUE THREAD

**Issue by frankhetterich** [marcinbudny/eventstore_exporter#19]
Hi,
we want to use the eventstore_exporter with a config file to inject credentials from a vault into the container. 
I can't quite figure out from the linked namsral/flag. 
How do i configure the eventstore_exporter to use a config file? Or is there a default path and filename?

Any help is appreciated

regards,
Frank

**Comment by marcinbudny** [comment]
Hi Frank,
This is the first time someone tries to use file based config, and I think some code might be missing to actually load the file. I will look into this.

**Comment by marcinbudny** [comment]
This is now fixed in version 0.10.2.

Sample configuration file

```
eventstore-url=http://localhost:2113
port=8888
verbose
```

To run with configuration file:

```
./eventstore_exporter --config my_config_file
```
